Karl/render.h

```diff
@@ -137,54 +137,53 @@ inline graphviz::DiGraph Render(const current::karl::GenericKarlStatus<INNER_STA
 
       // Second section, build info.
       {
-        auto text = service.git_commit;
-        if (text.empty()) {
-          text = "build";
-        } else if (text.length() > 6) {
-          text = text.substr(0, 6);
-        }
+        const std::string commit_text = Exists(service.git_commit) ? Value(service.git_commit).substr(0, 6) : "build";
```

Comment:
`"build"` reads confusing. The message is that it's out-of-git, manual, build, right?